share/spice/currency/currency.js

```diff
@@ -287,13 +287,15 @@
             }
 
             var endpoint = "/js/spice/currency/1/" + from + "/" + to;
-            $.get(endpoint, function(payload) {
-
-                // jsonp is returned from the API so we have to alter the contents
-                var response = JSON.parse(payload.trim().replace(/^[^\(]*\(/, '').replace(/\);$/, ''));
-                response = response.conversion;
-                Converter.resetConverter(response);
-
+            $.ajax({ 
+                url: endpoint,
+                dataType: 'text',
+                success: function(payload) {
+                    // jsonp is returned from the API so we have to alter the contents
```

Comment:
JavaScript is returned.

Reply:
Will remove -- I just copied the code as it was.